src/com/lftechnology/task/jan07/FileInput.java

```diff
@@ -0,0 +1,108 @@
+package com.lftechnology.task.jan07;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+/**
+ * Q4. Write a program called FileInput to read an int, a double, and a String form a text file called "in.txt", and produce the following
+ * output: (Hint use Scanner to read from file) a. The integer read is 12 b. The floating point number read is 33.44 c. The String read is
+ * "Peter" d. Hi! Peter, the sum of 12 and 33.44 is 45.44
+ * 
+ * This class reads the files and then lists out all its words with numbers and strings. Then it adds up all the numbers and finally
+ * displays in the given format.
+ * <p>
+ * However if there is no file location in the path, then a empty file is created.
+ * 
+ * Eg.the file have contents "ram shyam 10 10 10 34.5"
+ * 
+ * then the result will be Hi!ram, shyam the sum of 10.0, 10.0, 10.0 and 34.5 is 64.5.
+ * 
+ * @author Sanjib<sanjibmaharjan@lftechnology.com>
+ * 
+ */
+public class FileInput {
+	public static final Logger LOGGER = Logger.getLogger(FileInput.class.getName());
+
+	public static void main(String[] args) {
+		ArrayList<String> fileWordLists = new ArrayList<String>();
```

Comment:
Follow the program to interface design.
ArrayList<String> fileWordLists = new ArrayList<String>();
should be
List<String> fileWordLists = new ArrayList<String>();
http://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n
http://www.quora.com/Difference-between-both-the-List-and-ArrayList-declarations